src/apps/monero/diag.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Author: Dusan Klinec, ph4r05, 2018
+
+import gc
+import micropython
+import sys
+
+from trezor import log
+
```

Comment:
Same here, can we either get rid of this module or wrap it in `__debug__`? Is is useful to have it in production?

Reply:
It is definitely not needed for the production. I used it quite often during the development process, e.g., to track memory consumption, memory leaks, heap fragmentation. So for testing / development it is nice to have. Depends what you prefer, whether to remove it from the code completely or wrap in `__debug__` branch.